package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Lol I feel like we bump major versions often now

Reply:
🤷🏽 yeahh technically for semver, this is right, it's like a new backwards compatible feature/addition -- patch is usually for bug fixes

Reply:
This is a _minor_ version, not major, and it's accurate 😛

Reply:
Ok fair but we used to do like 1.4.1

Reply:
But I agree this is the right way to do it